requirements.txt

```
@@ -4,3 +4,4 @@ black==22.8.0
 Django==4.1.1
 python-dotenv==0.21.0
 psycopg2-binary==2.9.3
+django-bootstrap5
```

Can we lock the version here, like we do for the others? Makes dependency management more predictable.
```suggestion
django-bootstrap5==22.1
```

---

right